app/javascript/bundles/Admin/components/Admin.jsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import axios from 'axios';
+import GoogleMapReact from 'google-map-react';
+import Geocoder from 'react-native-geocoding';
+import './Admin.css';
+
+Geocoder.init('AIzaSyDxCfoKfjZjLCMX2gPTs2D8NfE1uGNXBac');
+
+const Marker = (props) => {
+    return (
+      <button className="pin"/>
+    );
+  };
+
+const RoomType = (props) => {
+    return (
+      <div>
+        <div className="form-group">
+          <label htmlFor="name">Type</label>
+          <select id={"type"+props.index}>
```

Comment:
you can use the syntax of es6 here:
``type${ props.index }``
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Done